app/app.js

```diff
@@ -0,0 +1,43 @@
+// app.js
+
+
+const express = require('express');
+
+
+ /* Importing the controller files */
+ const {getTopics} = require('../controllers/topics.controller');
+
+
+ /* Middleware functions */
+const app = express();
+app.use(express.json())
+
+
+ /* API calls */
+ app.get('/api/topics', getTopics)
+
+
+
+// custom errors
+app.use((err, req, res, next) => {
+  if (err.code) {
+    next()
+    res.status(err.status).send({ msg: err.msg });
+  } else {
+    next(err);
+  }
+});
+
+
+app.use((err, req, res, next) => {
+    if (err.status) {
+      res.status(err.status).send({ msg: err.msg });
+    } else next(err);
+  });
+  
```

Comment:
The above 2 error handlers don't need to be here just yet. Only add them when you come to implementing the errors after testing for them. Otherwise, you may find your error tests will pass straight away!